docs/src/format/table/index.md

```diff
@@ -16,7 +16,137 @@ A `Lance Dataset` is organized in a directory.
 A `Manifest` file includes the metadata to describe a version of the dataset.
 
 ```protobuf
-%%% proto.message.Manifest %%%
+// Manifest is a global section shared between all the files.
```

Comment:
the protobuf related changes should be reverted, we want to reference the protobuf file instead of copying the content here